string username
---
int64 score